lumicks/pylake/correlated_stack.py

```diff
@@ -177,8 +177,13 @@ def plot(self, frame=0, channel="rgb", show_title=True, axes=None, **kwargs):
         default_kwargs = dict(cmap="gray", vmax=None)
         kwargs = {**default_kwargs, **kwargs}
 
-        image = self._get_frame(frame)._get_plot_data(channel, vmax=kwargs["vmax"])
-        axes.imshow(image, **kwargs)
+        image = self._get_frame(frame)._get_plot_data(
+            channel, vmax=kwargs["vmax"], adjustment=adjustment
+        )
```

Comment:
Unfortunately, it's not that simple because the `vmax` is implemented kind of weirdly in `_get_plot_data`.

Because of the way it's implemented, `vmax` for widefield is expressed in the space of `(self.data / (2**self.bit_depth - 1))`.

Personally, I'd vote for dropping it (raise with an error to use `ColorAdjustment` instead).